lua/default-config.lua

```diff
@@ -7,6 +7,7 @@ USER = vim.fn.expand "$USER"
 O = {
   leader_key = "space",
   colorscheme = "spacegray",
+  line_wrap_cursor_movement = false,
```

Comment:
can you set the default config to `true`, since this is the current configuration?

Reply:
I have done this, although I think it encourages poor practice in VIM - some people (https://www.youtube.com/watch?v=E-ZbrtoSuzw) even disable `hjkl` to train themselves to use more efficient movement!

Reply:
yeah I agree but a lot of LunarVim users are just starting to migrate to neovim, so we don't want to scare them away :sweat_smile: